Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def volume(outerRadius : int, innerRadius : int, height : int) -> float:
    """calculates the volume of a hollow cylinder based 
    on the radius of the outer and inner cylinder"""
    
    surfaceC1 : float = math.pi * outerRadius**2
    surfaceC2 : float = math.pi * innerRadius**2
    print('surfaceC1 is', surfaceC1)
    print('surfaceC2 is', surfaceC2)
    
    return 0.0

volume(3, 1, 4)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn [6], line 12[0m
[1;32m      8[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124msurfaceC2 is[39m[38;5;124m'[39m, surfaceC2)
[1;32m     10[0m     [38;5;28;01mreturn[39;00m [38;5;241m0.0[39m
[0;32m---> 12[0m volume([38;5;241m3[39m, [38;5;241m1[39m, [38;5;241m4[39m)

Cell [0;32mIn [6], line 5[0m, in [0;36mvolume[0;34m(outerRadius, innerRadius, height)[0m
[1;32m      1[0m [38;5;28;01mdef[39;00m [38;5;21mvolume[39m(outerRadius : [38;5;28mint[39m, innerRadius : [38;5;28mint[39m, height : [38;5;28mint[39m) [38;5;241m-[39m[38;5;241m>[39m [38;5;28mfloat[39m:
[1;32m      2[0m     [38;5;124;03m"""calculates the volume of a hollow cylinder based [39;00m
[1;32m      3[0m [38;5;124;03m    on the radius of the outer and inner cylinder"""[39;00m
[0;32m----> 5[0m     surfaceC1 : [38;5;28mfloat[39m [38;5;241m=[39m [43mmath[49m[38;5;241m.[39mpi [38;5;241m*[39m outerRadius[38;5;241m*[39m[38;5;241m*[39m[38;5;241m2[39m
[1;32m      6[0m     surfaceC2 : [38;5;28mfloat[39m [38;5;241m=[39m math[38;5;241m.[39mpi [38;5;241m*[39m innerRadius[38;5;241m*[39m[38;5;241m*[39m[38;5;241m2[39m
[1;32m      7[0m     [38;5;28mprint[39m([38;5;124m'[39m[38;5;124msurfaceC1 is[39m[38;5;124m'[39m, surfaceC1)

[0;31mNameError[0m: name 'math' is not defined
NameError: name 'math' is not defined

